descriptors/ModuleDescriptor-template.json

```diff
@@ -388,6 +388,26 @@
             "consortia.sharing-roles.item.delete"
           ],
           "modulePermissions": []
+        },
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/consortia/{consortiumId}/sharing/roles/capability-sets",
+          "permissionsRequired": [
+            "consortia.sharing-roles-capability-sets.item.post"
```

Comment:
need to add declaration of these 2 permissions in permissions section:
consortia.sharing-roles-capability-sets.item.post
consortia.sharing-roles-capability-sets.item.delete